views/component/search/template.njk

```diff
@@ -1,5 +1,6 @@
 <div class="search-container">
 	<form action="/content-management/search/" method="get">
+	    <label for="q">
 		<input class="govuk-input search" id="q" type="text" name="q" value="{{query}}" placeholder="Search" />
```

Comment:
I would name this element a bit more meaningful 'q -> searchInput'

Reply:
that's how it was for the input. I didn't want to change it as I thought that it might break something else. So, I just kept it. Shall I rename it?